<a href="https://colab.research.google.com/github/argosmaia/ArritmiaDataScience/blob/main/trabalho.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **Questão 1 - a) Descreva a base de dados**

In [1]:
from scipy.io import arff
import pandas as pd

In [3]:
caminho = "/content/arritmias_treino.arff"

dados = arff.loadarff(caminho)
base = pd.DataFrame(dados[0])
base

Age   Sex  Height  Weight  QRS_duration    P-R    Q-T  T_interval  \
0    55.0  b'0'   175.0    94.0         100.0  202.0  380.0       179.0   
1    40.0  b'1'   160.0    52.0          77.0  129.0  377.0       133.0   
2    49.0  b'1'   162.0    54.0          78.0    0.0  376.0       157.0   
3    44.0  b'0'   168.0    56.0          84.0  118.0  354.0       160.0   
4    45.0  b'1'   165.0    86.0          77.0  143.0  373.0       150.0   
..    ...   ...     ...     ...           ...    ...    ...         ...   
397  65.0  b'0'   172.0    66.0          98.0  199.0  339.0       149.0   
398  50.0  b'1'   163.0    65.0          75.0  151.0  362.0       137.0   
399  27.0  b'1'   160.0    42.0          78.0  153.0  374.0       165.0   
400  58.0  b'1'   160.0    90.0          82.0    0.0  284.0       196.0   
401  63.0  b'1'   167.0    68.0          94.0  175.0  439.0       139.0   

     P_interval   QRS  ...  Amp_V6_2  Amp_V6_3  Amp_V6_4  Amp_V6_5  Amp_V6_6  \
0         143.0  28.0  ...       0.0      12.2      -2.2       0.0       0.0   
1          77.0  77.0  ...       0.0       6.5       0.0       0.0       0.0   
2          70.0  67.0  ...       0.0       8.2      -1.9       0.0       0.0   
3          63.0  61.0  ...       0.0       7.0      -1.3       0.0       0.0   
4          65.0  12.0  ...       0.0       4.4      -2.2       0.0       0.0   
..          ...   ...  ...       ...       ...       ...       ...       ...   
397       181.0  -4.0  ...       0.0      15.0      -4.5       0.0       0.0   
398        93.0  43.0  ...      -0.4       5.1      -1.3       0.0       0.0   
399        85.0  23.0  ...      -0.7      12.2      -2.4       0.0       0.0   
400         0.0  56.0  ...       0.0      14.5      -2.3       0.0       0.0   
401       100.0  25.0  ...       0.0       7.6      -2.2       0.0       0.0   

     Amp_V6_7  Amp_V6_8  Amp_V6_9  Amp_V6_10  class  
0         0.4       2.6      34.6       61.6   b'1'  
1         0.4       1.0      14.3       20.5   b'1'  
2         0.1       0.5      15.8       19.8   b'1'  
3         0.6       2.1      12.5       30.9   b'1'  
4         0.5       1.5       4.9       17.2   b'1'  
..        ...       ...       ...        ...    ...  
397       0.2      -0.4      26.4       23.0  b'10'  
398       0.7       1.0       6.2       11.6   b'1'  
399       0.5       1.9      11.0       25.0   b'1'  
400       0.2      -1.7      30.2       16.0  b'15'  
401       0.8       0.4      14.9       17.1   b'1'  

[402 rows x 280 columns]

**Age**: *Numérico*(razão) = Mostra a idade dos 402 pacientes presentes na lista

**Sex**: *Categórico* (binário) - Gênero do paciente (0: Masculino, 1: Feminino) = Mostra o sexo do paciente

**Height**: *Numérico* (razão) = Mostra a altura dos pacientes

**Weight**: *Numérico* (razão) = Mostra o peso dos pacientes

**QRS_duration**: *Numérico* (razão) = Duração do complexo QRS

**P-R**: *Numérico* (razão) = intervalo P-R

**Q-T**: *Numérico* (razão) = Intervalo Q-T

**T_interval**: *Numérico* (razão) = intervalo T

**P_interval**: *Numérico* (razão) = intervalo P

**QRS**: *Numérico* (razão) = valor do complexo de QRS

**Categoria numérica de J até Amp_V6_10**: Representando valores de amplitude de canais diferentes de ESG

**Class**: Categórico (nominal) = Classe de dados de arritmia cardíaca

